docs/blogs/elasticdl-antfin-introduction.md

```diff
@@ -104,6 +104,8 @@ ElasticDL 的 master 给 worker 分派计算任务 task 的示意图：
 
 ![AUC with different workers](../images/auc_with_different_workers.jpg)
 
+ 此实验使用了异步SGD更新。多项研究(如[[Ho2013](http://www.cs.cmu.edu/~seunghak/SSPTable_NIPS2013.pdf)], [[Zhang2016](https://arxiv.org/pdf/1511.05950.pdf)])表明，在异步SGD下worker数量越多，参数滞后(Staleness)越严重，并且严重的staleness会影响到模型收敛。在ElasticDL弹性调度下，worker数量可能因为被抢占而下降，staleness会减轻，理论上不会对模型训练产生负面影响。实验结果也符合此理论。
```

Comment:
Add white spaces between Chinese and English words.

Reply:
Added.